src/extension/mixer.ts

```diff
@@ -0,0 +1,162 @@
+import OSC from 'osc-js';
+import { get } from './util/nodecg';
+import { TaggedLogger } from './util/tagged-logger';
+import { Channel } from 'src/types/custom';
+
+const nodecg = get();
+
+const config = nodecg.bundleConfig.mixer;
+const log = new TaggedLogger('mixer');
+const channelIdToName = {
+  "1": "H1",
+  "2": "H2",
+  "3": "H3",
+  "4": "H4",
+  "9": "Host1",
+  "10": "Host2",
+  "16": "Host3",
+  "5": "Gra1",
+  "7": "Gra2",
+  "11": "Donacje",
+};
+const channelNameToId = Object.assign({}, ...Object.entries(channelIdToName).map(([k, v]) => ({ [v]: k })))
+const mixerSignalLevels = nodecg.Replicant<{ [key in Channel]: number }>('mixerSignalLevels', { defaultValue: {
+  "": -Infinity,
+  "H1": -Infinity,
+  "H2": -Infinity,
+  "H3": -Infinity,
+  "H4": -Infinity,
+  "Host1": -Infinity,
+  "Host2": -Infinity,
+  "Host3": -Infinity,
+  "Gra1": -Infinity,
+  "Gra2": -Infinity,
+  "Donacje": -Infinity,
+}});
+
+// TODO add some panel to control them in case we need to tweak them live
+const mixerThresholdLevels = nodecg.Replicant<{ [key in Channel]: number }>('mixerThresholdLevels', { defaultValue: {
+  "": -Infinity,
+  "H1": -30,
+  "H2": -30,
+  "H3": -30,
+  "H4": -30,
+  "Host1": -30,
+  "Host2": -30,
+  "Host3": -30,
+  "Gra1": -30,
+  "Gra2": -30,
+  "Donacje": -30,
+}});
```

Comment:
```suggestion
const channelIdToName = {
  "1": "H1",
  "2": "H2",
  "3": "H3",
  "4": "H4",
  "9": "Host1",
  "10": "Host2",
  "16": "Host3",
  "5": "Gra1",
  "7": "Gra2",
  "11": "Donacje",
};
const channelNameToId = Object.entries(channelIdToName).reduce((acc, [id, name]) => {
  acc[name] = id;
  return acc;
}, {} as Record<string, string>);
const mixerSignalLevels = nodecg.Replicant<{ [key in keyof typeof channelNameToId]: number }>('mixerSignalLevels', {
  defaultValue: Object.keys(channelNameToId).reduce((acc, name) => {
    acc[name] = -Infinity
    return acc;
  }, {"": -Infinity} as Record<string, number>)
});

// TODO add some panel to control them in case we need to tweak them live
const mixerThresholdLevels = nodecg.Replicant<{ [key in keyof typeof channelNameToId]: number }>('mixerThresholdLevels', {
  defaultValue: Object.keys(channelNameToId).reduce((acc, name) => {
    acc[name] = -30
    return acc;
  }, {"": -Infinity} as Record<string, number>)
});
```

Maybe do replicant generation like this?

Reply:
Looks much cleaner now, thanks